src/pods/canvas/canvas.business.ts

```diff
@@ -73,3 +79,75 @@ export const calculateRelationXCoordinate = (
   xOrigin: calculateRelationXCoordinateOrigin(tableOrigin, tableDestination),
   xDestination: calculateRelationXCoordinateEnd(tableOrigin, tableDestination),
 });
+
+const seekField = (
+  fieldId: GUID,
+  YPosition: number,
+  fields: FieldVm[]
+): seekResult => {
+  let found = false;
+  let parentCollapsed = false;
+  let newYPosition = YPosition;
+
+  for (let i = 0; i < fields.length && !found; i++) {
+    const field = fields[i];
+    if (field.id === fieldId) {
+      found = true;
```

Comment:
Since we are just returning a value we can get rid of 

```diff
- found = true:
```

And pass `found: true` in the object it self

Reply:
It's fixed